department-of-veterans-affairs/vets-website
# VAGOV-4546: Facility | Add Service ID field

## Description
refactors the appointment wait time widget to use a different, much more reliable, field to match health service api names

## Testing done
locally with staging data

pre-req: need to populate "Health Service API ID" on the CMS side for Health Care Taxonomies
1. go to a health care taxonomy such as "Primary care" http://staging.cms.va.gov/taxonomy/term/37/edit
2. edit
3. fill in the "Health Service API ID" with "primaryCare"
4. save

To test:
1. get latest and rebuild site
2. go to a page such as https://staging.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/#health-care-offered-here
3. open up the "Primary care" accordion
4. the "Patient wait times" should be there


## Screenshots
![image](https://user-images.githubusercontent.com/19178435/62173984-04db8780-b2ed-11e9-8d17-c4da1b7e0bec.png)

![image](https://user-images.githubusercontent.com/19178435/62174025-276da080-b2ed-11e9-8092-4b688f607a5c.png)

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-4546

## Definition of done
